app/experimenter/experiments/api/v5/inputs.py

```diff
@@ -0,0 +1,20 @@
+import graphene
+
+
+class CreateExperimentInput(graphene.InputObjectType):
+    clientMutationId = graphene.String()
+    name = graphene.String(required=True)
+    slug = graphene.String(required=True)
+    application = graphene.String()
+    public_description = graphene.String()
+    hypothesis = graphene.String()
+
+
+class UpdateExperimentInput(graphene.InputObjectType):
+    id = graphene.ID(required=True)
+    clientMutationId = graphene.String()
+    name = graphene.String(required=True)
+    slug = graphene.String(required=True)
+    application = graphene.String()
+    public_description = graphene.String()
+    hypothesis = graphene.String()
```

Comment:
Can you subclass the create one and just add id?

Reply:
Probably, though I just noticed this is actually wrong, the update does *not* require the same fields as the creation.

Reply:
Well in either case if one is a superset of the other you can probably just subclass?